modules/ROOT/pages/dataweave-cookbook-merge-multiple-payloads.adoc

```diff
@@ -1,78 +1,23 @@
-= Merge Multiple Payloads
+= Merge Fields from Separate Objects
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, format, aggregate, filter, json, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping, contains, as
 
-This DataWeave example deals with an input that contains a collection of collections. The first payload index `payload[0]` lists price by book, and the second `payload[1]` lists authors by book, where each book can be identified by a unique `bookId` key. This transformation merges both payload array elements into one, matching their keys, where one book may have several authors.
+The DataWeave examples merge fields from separate input arrays. The first   (`firstInput`) is a DataWeave variable that lists price by book ID, and the second (`secondInput`) lists authors by book ID. Each book has a unique `bookId` key.
 
+The DataWeave scripts produce the same output. Both scripts use two `map` functions with a filter, and one of them also creates an alias for the `bookId`:
```

Comment:
Need a new line after this (to see it nicely in github)

Reply:
fixed